docs/conf/portland/2021/writing-day.rst

```diff
@@ -49,6 +49,28 @@ Write the Docs Meetups
 
 Organizing local Write the Docs meetup communities is a rewarding way to participate. During Writing Day, we'll have a table where we can share tips and best practices, especially in this time where all of our meetups are virtual.
 
+GitLab - Learn how to contribute to open-source docs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`GitLab <https://about.gitlab.com>`_ is a single application for
+the entire software development lifecycle. From project planning and source code
+management to CI/CD, monitoring, and security.
+
+Our `documentation <https://docs.gitlab.com>`_ is open source and everybody can
+contribute to it. All the content is hosted on GitLab.com, so you will need a
+`GitLab.com account <https://gitlab.com/users/sign_up>`_.
+
+For more information, see our `GitLab issue <https://gitlab.com/gitlab-org/technical-writing/-/issues/426>`_.
+
+Once you find the issue you want to work on, add a comment mentioning `@gl-docsteam` to inform us that you’re working on
+   this for Writing Day (and tell us in person!).
```

Comment:
The sphinx error is from this sudden indent, this makes it a [definition list](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks) for sphinx - which needs a blank line after. I think the definition list syntax is unintentional though. I don't know what the line after is supposed to look like, so I'm leaving to you to edit it.

Reply:
Thank you @mxsasha . Hopefully the tests will pass after I remove the rogue indent.